src/common.py

```diff
@@ -88,7 +89,12 @@ def verify_admin(netid, cid):
 	:param cid: a course ID.
 	:return: a boolean value.
 	"""
-	return netid in db.get_course(cid)["admin_ids"]
+	staff = db.get_course(cid)["staff"]
+	if netid not in staff:
+		return False
+	else:
+		is_admin = staff[netid]
+		return is_admin == {"is_admin": True}
```

Comment:
If every staff has a dictionary, we can simplify this to
```
return staff[netid]["is_admin"]
```
Also in case we will add new fields other than `is_admin` to staffs in the future.